### ISSUE TITLE
Write integration tests for JsDbg

### ISSUE THREAD

**Issue by cbiesinger** [MicrosoftEdge/JsDbg#180]
Somewhat related to #178  - there are currently no tests for the debugger/webserver integration. We could write a small test program to be debugged, launch the debugger and jsdbg with it, and make requests to the web server to ensure we get the right answers.

This should be fairly straightforward for GDB, at least; for Windbg this might be doable using remote debugging perhaps?

**Comment by sanketj** [comment]
Yes, makes sense and should be doable for WinDbg. For VS, I'm unsure on the right way to do this, but hopefully we can figure it out.
